sdk/typespec/typespec_macros/README.md

```diff
@@ -31,11 +31,23 @@ struct Credentials {
   pub username: String,
   pub password: String,
 };
-let credentials: Credentials = Credentials {
+
+let credentials = Credentials {
   username: "admin".into(),
   password: "hunter2".into(),
```

Comment:
The code scanners haven't fired for a checked in pwd yet, but I wonder if they will?

Reply:
But it's already masked. How do you even see it?! :wink:

We can cross that bridge if we get there. I would hope, for samples, they have a list of "allowed" and I'd be shocked - horrified, even - if "hunter2" wasn't on there.